src/test/java/de/skuzzle/enforcer/restrictimports/impl/ImportMatcherImplTest.java

```diff
@@ -124,6 +124,28 @@ public void testExcludeFile() throws Exception {
         assertFalse(matches.iterator().hasNext());
     }
 
+    @Test
```

Comment:
Another test case with some inline comments or maybe a mix of block and inline comments would be great